$NetBSD: patch-ab,v 1.1 2010/08/20 19:15:28 sno Exp $

Patch to switch from Wx::Perl::Dialog (Padre-0.66) to Padre::Wx::Dialog

--- lib/Padre/Plugin/Vi/CommandLine.pm.orig	2010-08-20 19:07:14.000000000 +0000
+++ lib/Padre/Plugin/Vi/CommandLine.pm
@@ -3,11 +3,11 @@ package Padre::Plugin::Vi::CommandLine;
 use warnings;
 use strict;
 
-use Cwd              ();
-use Wx::Perl::Dialog ();
-use Padre::Wx        ();
-use File::Spec       ();
-use File::Basename   ();
+use Cwd               ();
+use Padre::Wx::Dialog ();
+use Padre::Wx         ();
+use File::Spec        ();
+use File::Basename    ();
 
 use Padre::Plugin::Vi::TabCompletition qw(clear_tab handle_tab set_original_cwd);
 
@@ -66,7 +66,7 @@ sub dialog {
 
 	my $main = Padre->ide->wx->main;
 	if ( not $dialog ) {
-		$dialog = Wx::Perl::Dialog->new(
+		$dialog = Padre::Wx::Dialog->new(
 			parent => $main->{notebook},
 			title  => "Command Line",
 			layout => \@layout,
